connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -82,7 +82,7 @@
     void connectorConfig(String connName, Callback<Map<String, String>> callback);
 
     /**
-     * Set the configuration for a connector. This supports creation, update, and deletion.
+     * Set the configuration for a connector. This supports creation and updating.
```

Comment:
No, I was worried that previously we handled the `null` as a deletion, so `ConnectorsResource.putConnectorConfig()` only calling `Herder.putConnectorConfig()` may have changed behavior -- previously with DistributedHerder the deletion would have worked whereas now it will throw an exception.